vendor/ehs/httpresponse.h

```diff
@@ -17,14 +17,23 @@
 #include "datum.h"
 
 /// different response codes and their corresponding phrases -- defined in EHS.cpp
-enum ResponseCode { HTTPRESPONSECODE_INVALID = 0,
-					HTTPRESPONSECODE_200_OK = 200,
-					HTTPRESPONSECODE_301_MOVEDPERMANENTLY = 301,
-					HTTPRESPONSECODE_302_FOUND = 302,
-					HTTPRESPONSECODE_401_UNAUTHORIZED = 401,
-					HTTPRESPONSECODE_403_FORBIDDEN = 403,
-					HTTPRESPONSECODE_404_NOTFOUND = 404,
-					HTTPRESPONSECODE_500_INTERNALSERVERERROR = 500 };
+enum ResponseCode {
```

Comment:
Not sure if modifying EHS vendor code is a good idea.
Why not use these HTTP codes directly?

Reply:
The return type is ResponseCode. I wanted to return that instead of ints instead